### PR TITLE
Update ioping yaml

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Add description for `ioping`  plugin.

##### Test Plan

1. Take a look in CI and grammar.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? documentation, integration and dashboard.
- Can they see the change or is it an under the hood? If they can see it, where? no
- How is the user impacted by the change? a plugin well documented.
- What are there any benefits of the change? A more clear information for our users.
</details>

### CODE REVIEW

File: collectors/ioping.plugin/metadata.yaml [hunk 1]
```diff
@@ -33,17 +33,38 @@ overview:
       description: ''
 setup:
   prerequisites:
-    list: []
+    list: |
+      It is necessary to have the command `ioping` installed on host. It is possible to install command passing as argument for plugin the option `install`:
```
Comment (sashwathn) [inline collectors/ioping.plugin/metadata.yaml:37]: It is necessary to have the command `ioping` installed on your host. You can install the command by passing the argument `install` to the plugin:

**Review by sashwathn — DISMISSED** [review]

File: collectors/ioping.plugin/metadata.yaml (hunk 1 above)
Comment (thiagoftsm) [reply collectors/ioping.plugin/metadata.yaml]: Thank you @sashwathn , I push a commit with your suggestion.

File: collectors/ioping.plugin/metadata.yaml [hunk 2]
```diff
@@ -33,17 +33,38 @@ overview:
...
+      list: |
+        - name: update_every
+          description: Data collection frequency.
+          default_value: 1s
+          required: false
+        - name: ioping
+          description: The complete plugin path.
+          default_value: /usr/libexec/netdata/plugins.d/ioping
+          required: false
+        - name: request_size
```
Comment (Ancairon) [inline collectors/ioping.plugin/metadata.yaml:60]:
```suggestion
	- name: destination
	  description: The directory/file/device to ioping.
	  default_value: ""
	  required: true
        - name: request_size
```

File: collectors/ioping.plugin/metadata.yaml [hunk 3]
```diff
@@ -33,17 +33,38 @@ overview:
...
+          description: The request size in bytes to ioping the destination (symbolic modifiers are supported)
+          default_value: 4k
+          required: false
+        - name: ioping_options
```
Comment (Ancairon) [inline collectors/ioping.plugin/metadata.yaml:64]: I think conf file has it as ioping_opts
```suggestion
        - name: ioping_opts
```

File: collectors/ioping.plugin/metadata.yaml [hunk 4]
```diff
@@ -33,17 +33,38 @@ overview:
...
+          description: Options passed to `ioping` commands.
+          default_value: -T 1000000
+          required: false
     examples:
```
Comment (Ancairon) [inline collectors/ioping.plugin/metadata.yaml:68]: @thiagoftsm @ilyam8 @Ferroin I don't know who would be best to provide a configuration example, but we need one here, and I don't know what destination should be (I guess something like /dev/sda)

Comment (thiagoftsm) [reply collectors/ioping.plugin/metadata.yaml:68]: We only need to modify one parameter inside `ioping.conf`, so I pushed the minimum example and I also address other comments in last commit.

File: collectors/ioping.plugin/metadata.yaml [hunk 5]
```diff
@@ -33,22 +33,44 @@ overview:
...
+      name: 'ioping.conf'
+      description: 'File with options to specify hardware to monitor and arguments for ioping command.'
     options:
       description: ''
       folding:
-        title: ''
+        title: 'Config options'
         enabled: true
-      list: []
+      list: |
```
Comment (andrewm4894) [inline collectors/ioping.plugin/metadata.yaml:51]:
```suggestion
      list:
```

File: collectors/ioping.plugin/metadata.yaml [hunk 6]
```diff
@@ -33,22 +33,44 @@ overview:
...
+          description: Options passed to `ioping` commands.
+          default_value: -T 1000000
+          required: false
     examples:
       folding:
         enabled: true
         title: ''
-      list: []
+      list: |
+        destination="/dev/sda"
```
Comment (andrewm4894) [inline collectors/ioping.plugin/metadata.yaml:73]:
```suggestion
      list:
        - destination="/dev/sda"
```

File: collectors/ioping.plugin/metadata.yaml [hunk 7]
```diff
@@ -33,22 +33,44 @@ overview:
...
+        - name: ioping_opts
+          description: Options passed to `ioping` commands.
+          default_value: -T 1000000
+          required: false
     examples:
       folding:
         enabled: true
         title: ''
-      list: []
+      list: |
```
Comment (andrewm4894) [inline collectors/ioping.plugin/metadata.yaml:72]: i am not sure what this is - if you want to do an example i think needs to be something like this: https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/example/metadata.yaml#L98

File: collectors/ioping.plugin/metadata.yaml [hunk 8]
```diff
@@ -53,23 +53,24 @@ setup:
...
           description: Options passed to `ioping` commands.
           default_value: -T 1000000
           required: false
     examples:
       folding:
         enabled: true
         title: ''
-      list: []
+      list: |
+        destination="/dev/sda"
```
Comment (Ancairon) [inline collectors/ioping.plugin/metadata.yaml:73]: @thiagoftsm please use the schema in order to fill this in, this is not the way we define examples, check https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/dovecot/metadata.yaml#L93-L110

Comment (Ancairon) [reply collectors/ioping.plugin/metadata.yaml:73]: ah, Andy said the same thing, sorry for this

**Review by sashwathn — APPROVED** [review]

**Comment by Ancairon** [comment]
@sashwathn this is not good to merge, it fails schema validation

File: collectors/ioping.plugin/metadata.yaml (hunk 8 above)
Comment (thiagoftsm) [reply collectors/ioping.plugin/metadata.yaml]: Done

File: collectors/ioping.plugin/metadata.yaml [hunk 9]
```diff
@@ -1,54 +1,81 @@
...
-      description: ''
+      description: ""
     performance_impact:
-      description: ''
+      description: ""
 setup:
   prerequisites:
-    list: []
+    list:
+      It is necessary to have the command `ioping` installed on your host. You can install the command by passing the argument `install` to the plugin:
```
Comment (ilyam8) [inline collectors/ioping.plugin/metadata.yaml:39]: @thiagoftsm you need configuring schema validation in your IDE. It is in "Languages & Frameworks"->"Schemas and DTDs"->"JSON Schema Mappings"

Comment (thiagoftsm) [reply collectors/ioping.plugin/metadata.yaml:39]: Vscode did not report issues at all.

File: collectors/ioping.plugin/metadata.yaml [hunk 10]
```diff
@@ -1,54 +1,79 @@
...
+          required: false
     examples:
       folding:
         enabled: true
-        title: ''
-      list: []
+        title: "Config"
+      list:
+        - name: Basic Configuration
+          description: This Examplle has the minimum configuration necessary to have plugin running.
```
Comment (shyamvalsan) [inline collectors/ioping.plugin/metadata.yaml:74]:
```suggestion
          description: This example has the minimum configuration necessary to have the plugin running.
```

**Review by shyamvalsan — CHANGES_REQUESTED** [review]

**Review by shyamvalsan — APPROVED** [review]

**Review by sashwathn — APPROVED** [review]

**Pull request merged**